project/plugins.sbt

```diff
@@ -1,11 +1,17 @@
+enablePlugins(BuildInfoPlugin)
+
+val sbtProtocV = "0.99.26"
+
+buildInfoKeys := Seq[BuildInfoKey]("sbtProtocVersion" -> sbtProtocV)
+
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.3.0")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.4.0")
-addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.19")
+addSbtPlugin("com.thesamet" % "sbt-protoc" % sbtProtocV)
 addSbtPlugin("com.dwijnand" % "sbt-dynver" % "4.0.0")
 addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.5.0")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```

Comment:
Is dependency graph used in the build or just here for convenience looking at dependencies (if so does it belong in the build vs a local build config, maybe skip it instead and keep the build lean)